alpine/Dockerfile

```diff
@@ -15,15 +15,13 @@ RUN apt-get update && \
 
 FROM alpine:3
 LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
-LABEL autoheal=true
 
 ARG SIA_DIR="/sia"
 ARG SIA_DATA_DIR="/sia-data"
 ARG SIAD_DATA_DIR="/sia-data"
 
 RUN mkdir /lib64 && \
-    ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2 && \
-    apk add --no-cache socat
+    ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
what are those for btw?

Reply:
Our go code is not entirely statically linked - it still needs `glibc` and it is looking for it under the name of `/lib64/ld-linux-x86-64.so.2`. Alpine is using `musl` instead of `glibc` but they are compatible, so we're just linking the lib to where `siad` will be looking for it.